tests/specs/scss/functions.scss

```diff
@@ -1,73 +1,73 @@
 
-@include test-module('Functions') {
+@include test-module("Functions") {
 
   // Testing set functions
-  @include test('config-set [function] Set a single key.') {
-    @include config-set('bar', 'foo');
-    @include config-set('color', #fff);
+  @include test("config-set [function] Set a single key.") {
+    @include config-set("bar", "foo");
+    @include config-set("color", #fff);
 
     $expect: ("bar": "foo", "color": #fff);
 
-    @include assert-equal($config-attr, $expect, 'Check if map is correct.');
+    @include assert-equal($config-attr, $expect, "Check if map is correct.");
   }
 
   // Testing set functions
-  @include test('config-set [function] Set a single !default key.') {
-    @include config-set('single-default', 'foo', true);
+  @include test("config-set [function] Set a single !default key.") {
+    @include config-set("single-default", "foo", true);
 
     $expect: ("single-default": "foo");
-    @include assert-equal($config-default, $expect, 'Check if map is correct.');
+    @include assert-equal($config-default, $expect, "Check if map is correct.");
   }
 
   // Testing set functions
-  @include test('config-set [function] Set a dotted key.') {
-    @include config-set('test.bar', 'foo');
-    @include config-set('test.color', #fff);
+  @include test("config-set [function] Set a dotted key.") {
+    @include config-set("test.bar", "foo");
+    @include config-set("test.color", #fff);
 
-    $expect: ("bar": 'foo', "color": #fff);
-    $config: config-get('test');
+    $expect: ("bar": "foo", "color": #fff);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.